### ISSUE TITLE
Hashed assets not working in documentation site

### ISSUE THREAD

**Issue by davidalber** [redfin/react-server#452]
With hashed assets enabled, the documentation site is not working. The site requests assets with a different hash than are generated by the earlier `build-assets` step. Here's what the site looks like with hashed assets:
![image](https://cloud.githubusercontent.com/assets/933552/17173536/3e9baabc-53b0-11e6-98c3-5e5bff9bbf78.png)

Here are the failing asset requests (plus another unrelated issue):
![image](https://cloud.githubusercontent.com/assets/933552/17173520/2b520564-53b0-11e6-8368-b5a628858a38.png)

_Hashed assets were working earlier in the project, so something changed to break them._

**Comment by gigabo** [comment]
Do we just need to pass `--js-url /assets/` in `build-assets`?

**Comment by davidalber** [comment]
Seemingly not. I tried using the following patch in react-server-website/package.json.

```diff
@@ -5,8 +5,8 @@
   "main": "HelloWorld.js",
   "private": true,
   "scripts": {
-    "build-assets": "npm run docs && node ./cli.js --compile-only --minify",
-    "start-prod": "node ./cli.js --port 3010 --minify --js-url /assets/",
+    "build-assets": "npm run docs && node ./cli.js --compile-only --minify --long-term-caching --js-url /assets/",
+    "start-prod": "node ./cli.js --port 3010 --minify --long-term-caching --js-url /assets/",
     "start": "node ./cli.js --port 3010 --js-port 3011",
     "test": "eslint routes.js gulpfile.js pages/ components/ && nsp check",
     "docs": "docco -o docs ./components/*.js ./lib/*.js ./middleware/*.js ./pages/*.js *.js && node build-dir-contents.js"
```

The same issue occurred.

**Comment by davidalber** [comment]
This was resolved by #517.
